workers/unity/Packages/com.improbable.gdk.core/Utility/EntityBuilder.cs

```diff
@@ -61,6 +85,14 @@ public EntityBuilder AddPosition(double x, double y, double z, string writeAcces
             return AddComponent(new ComponentData(schemaData), writeAccess);
         }
 
+        /// <summary>
+        ///     Sets the Persistence component to the entity template.
+        /// </summary>
+        /// <remarks>
+        ///    Persistence denotes whether the entity will be included in snapshots.
```

Comment:
Shall we mention it HAS to be used for entities going into snapshots? Or perhaps we could note that in snapshot generation instead (if we have not yet done so already)

Reply:
Yes we probably should - the worker SDK snapshot API will throw I think if you attempt to put an entity without persistence in there